docs/cluster_management.md

```diff
@@ -46,11 +46,11 @@ If you want more control over the upgrading process, you may use the following w
     This keeps new pods from landing on the node while you are trying to get them off.
   1. Get the pods off the machine, via any of the following strategies:
     1. wait for finite-duration pods to complete
-    1. for pods with a replication controller, delete the pod with `kubectl delete pods $PODNAME`
-    1. for pods which are not replicated, bring up a new copy of the pod, and redirect clients to it.
+    1. delete pods with `kubectl delete pods $PODNAME`
+    l. for pods with a replication controller, the pod will eventually be rescheduled to a new node. additionally, if the pod is part of a service, then clients will automatically be redirected to the new pod.
```

Comment:
You used lowercase L rather than 1.

Also, I'd say that the pod will be replaced and scheduled to a new node.